kafka-plugins-0.10/src/main/java/io/cdap/plugin/sink/KafkaBatchSink.java

```diff
@@ -223,14 +219,7 @@ public Config(String brokers, String async, String key, String topic, String for
     }
 
     private void validate(FailureCollector collector) {
-      if (useConnection()) {
-        collector.addFailure("Kafka batch sink plugin doesn't support using existing connection.",
-                             "Don't set useConnection property to true.");
-      }
```

Comment:
should the doc file be updated?

Reply:
For some reason, we forgot to remove these properties when we disabled them - https://github.com/data-integrations/kafka-plugins/blob/develop/kafka-plugins-0.10/docs/Kafka-batchsource.md#properties, https://github.com/data-integrations/kafka-plugins/blob/develop/kafka-plugins-0.10/docs/Kafka-batchsink.md#configuration